share/completions/keybase.fish

```diff
@@ -1,70 +1,156 @@
-#Keybase 5.0.0
+#Keybase 5.1.1
 
-function __fish_keybase_commandline_ends_with
-    set -l line (commandline -poc)
+function __fish_keybase_line_ends_with
+    set -l line
+    for s in (commandline -poc)
+        set -l c (string sub -l 1 -- $s)
+        if test "$c" != "-"; set -a line $s; end
```

Comment:
You can do this in one line:
`set -l line (commandline -poc | string match -v -r '^-')`

Reply:
I avoided regex for no reason, after a quick test it seems there is almost no difference.
That one line is much more elegant, thanks!